benches/select_numbers.rs

```diff
@@ -1,24 +1,29 @@
 use serde::Deserialize;
 
+use clickhouse::validation_mode::ValidationMode;
 use clickhouse::{Client, Compression, Row};
 
 #[derive(Row, Deserialize)]
 struct Data {
+    #[serde(rename = "number")]
```

Comment:
It seems that this rename is required now, so let's change the name of the field, because correctness should be checked in tests, not benchmarks.